apps/web/src/pages/Users/UserInformationPage/components/UserCandidatesTable/UserCandidatesTable.tsx

```diff
@@ -0,0 +1,261 @@
+import React from "react";
+import { ColumnDef, createColumnHelper } from "@tanstack/react-table";
+import { useIntl } from "react-intl";
+
+import {
+  Department,
+  FragmentType,
+  PoolCandidate,
+  UserInfoFragment,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getLanguage,
+  getLocalizedName,
+  getPoolCandidatePriorities,
+  getPoolCandidateStatus,
+} from "@gc-digital-talent/i18n";
+import { notEmpty, unpackMaybes } from "@gc-digital-talent/helpers";
+
+import Table from "~/components/Table/ResponsiveTable/ResponsiveTable";
+import { normalizedText } from "~/components/Table/sortingFns";
+import { getFullNameLabel } from "~/utils/nameUtils";
+import tableMessages from "~/components/PoolCandidatesTable/tableMessages";
+import useRoutes from "~/hooks/useRoutes";
+import { PoolCandidate_BookmarkFragment } from "~/components/CandidateBookmark/CandidateBookmark";
+import { getFullPoolTitleLabel } from "~/utils/poolUtils";
+import processMessages from "~/messages/processMessages";
+import adminMessages from "~/messages/adminMessages";
+import {
+  JobPlacementDialog_Fragment,
+  jobPlacementDialogAccessor,
+} from "~/components/PoolCandidatesTable/JobPlacementDialog";
+import cells from "~/components/Table/cells";
+import accessors from "~/components/Table/accessors";
+
+import {
+  bookmarkCell,
+  bookmarkHeader,
+  candidacyStatusAccessor,
+  candidateNameCell,
+  currentLocationAccessor,
+  finalDecisionCell,
+  notesCell,
+  priorityCell,
+  processCell,
+} from "./helpers";
+
+type PoolCandidateSlice = Pick<
+  PoolCandidate,
+  | "id"
+  | "user"
+  | "notes"
+  | "submittedAt"
+  | "status"
+  | "pool"
+  | "placedDepartment"
+  | "suspendedAt"
+  | "isBookmarked"
+>;
+
+const columnHelper = createColumnHelper<PoolCandidateSlice>();
+
+export interface UserCandidatesTableProps {
+  user: UserInfoFragment;
+  departments: Department[];
+  title: string;
+}
+
+const UserCandidatesTable = ({
+  user,
+  departments,
+  title,
+}: UserCandidatesTableProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const poolCandidatesUnpacked = user.poolCandidates?.filter(notEmpty) ?? [];
```

Comment:
This is exactly what `unpackMaybes` does so I think we should use that instead.

Reply:
Makes sense to me! https://github.com/GCTC-NTGC/gc-digital-talent/pull/10342/commits/7aa94c753ce15bc6c17a91d252d1811834aa5bf5